doc/encoding.rst

```diff
@@ -10,10 +10,22 @@ Encoding or vectorizing creates numerical features from the data,
 converting dataframes, strings, dates... Different encoders are suited
 for different types of data.
 
-.. _dirty_categories:
+Summary
+.......
+:class:`StringEncoder` should be used in most cases when working with high-cardinality
```

Comment:
StringEncoder is not rendered properly, not sure how to fix this

Reply:
you need to add it to the reference documentation index:

https://github.com/skrub-data/skrub/blob/main/doc/reference/index.rst?plain=1#L43


so that sphinx can create a page and a link for it